doc/plugin_agent_svidstore_aws_secretsmanager.md

```diff
@@ -0,0 +1,48 @@
+# Agent plugin: SVIDStore "aws_secretsmanager"
```

Comment:
We probably want to describe the JSON format that is used to store the SVID information? We may have a section in this document for that.